docs/modules/user/pages/try/15_minutes.adoc

```diff
@@ -0,0 +1,229 @@
+
+List all the current users:
+
+[source,bash]
+----
+./james listusers
+----
+
+You should see users ``user01@james.local``, ``user02@james.local``, and ``user03@james.local``.
+
+Create a new "test.local" domain:
+
+[source,bash]
+----
+./james adddomain test.local
+----
+
+List the domains again to ensure that "test.local" has successfully been added:
+
+[source,bash]
+----
+./james listdomains
+----
+
+Add the user "testuser" to the "test.local" domain with password "password":
+
+[source,bash]
+----
+./james adduser testuser@test.local password
+----
+
+You should now see your newly created user:
+
+[source,bash]
+----
+./james listusers
+----
+
+== Connect to the server with an email client
+
+****
+For this tutorial, we will use Thunderbird, as it is available in multiple languages
+on Windows, Mac, and Linux.
+
+Go to https://www.thunderbird.net to download Thunderbird.
+****
+
+After you have installed Thunderbird,
+https://support.mozilla.org/en-US/kb/manual-account-configuration[manually set up an account]
+for user01@james.local using the following parameters:
+
+ * Account name: user01
+ * Your name: User 01
+ * Email address: user01@james.local
+ * SMTP server name: localhost
+ * SMTP port: 25
+ * SMTP connection security: none
+ * SMTP authentication: password
+ * IMAP server name: localhost
+ * IMAP user name: user01@james.local
+ * IMAP port: 143
+ * IMAP connection security: none
+ * IMAP authentication: none
+
+Repeat the above for testuser@test.local:
+
+ * Account name: testuser
+ * Your name: Test User
+ * Email address: testuser@test.local
+ * SMTP server name: localhost
+ * SMTP port: 25
+ * SMTP connection security: none
+ * SMTP authentication: password
+ * IMAP server name: localhost
+ * IMAP user name: testuser@test.local
+ * IMAP port: 143
+ * IMAP connection security: none
+ * IMAP authentication: none
+
+Now have fun sending emails back and forth to yourself. :-)
+But don't spend all day doing this. You have too much work to do.
+
+== Connect to the server via the Admin API
+
+We will use curl to connect to localhost on port 8000, as the Admin API
+is made available on this port.
+
+List the available domains:
+
+[source,bash]
+----
+curl http://localhost:8000/domains
+----
+
+You should see a response similar to this:
+
+[source,json]
+----
+[
+  "edfce41c55e6",
+  "james.linagora.com",
+  "james.local",
+  "localhost",
+  "test.local",
+  "172.17.0.2"
+]
+----
+
+Test that a domain exists:
+
+[source,bash]
+----
+curl -I -X GET http://localhost:8000/domains/test.local
+----
+
+You should see an empty `204` response, which means "yes, this domain does exist".
+
+Delete our test domain:
+
+[source,bash]
+----
+curl -X DELETE http://localhost:8000/domains/test.local
+----
+
+Now retest `test.local`:
+
+[source,bash]
+----
+curl -I -X GET http://localhost:8000/domains/test.local
+----
+
+This time you will receive a `404` code because the `test.local` domain no longer exists.
```

Comment:
We could state that many more API are exposed via webadmin and are documented in detail https://james.apache.org/server/manage-webadmin.html , people can of course play longer with the content documented there ;-)

Reply:
Ok, good point.

Reply:
This is not yet documented in the new docs, so I included a placeholder instead.